validation/Cargo.toml

```diff
@@ -12,6 +12,7 @@ tokio = { version = "0.2.4", features = ["rt-core", "blocking"] }
 derive_more = "0.14.1"
 log = "0.4.8"
 exit-future = "0.2.0"
+tokio-executor = { version = "0.2.0-alpha.6", features = ["blocking"] }
```

Comment:
Uhh, an `alpha`? I hereby request the opinion of @tomaka on this one

Reply:
That's what it was before #673.

Reply:
I could revert further, but if it works, it works 🤷‍♀